packages/flutter/test/painting/mocks_for_image_cache.dart

```diff
@@ -9,7 +9,7 @@ import 'package:flutter/foundation.dart';
 import 'package:flutter/painting.dart';
 
 class TestImageInfo extends ImageInfo {
-  const TestImageInfo(this.value, {
+  TestImageInfo(this.value, {
```

Comment:
nit: this can still be const right?

Reply:
nope: `A constant constructor can't call a non-constant super constructor of 'TestImageInfo'.`